From 19eee1bdc76d9f3d8f2bef5c9786500fede11a59 Mon Sep 17 00:00:00 2001
From: Ronny Kumke <ronny@lxcars.de>
Date: Sun, 4 Mar 2012 18:20:09 +0100
Subject: [PATCH 2/2] stdlib

---
 inc/stdLib.php |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/inc/stdLib.php b/inc/stdLib.php
index a39bb8d..8f191be 100755
--- a/inc/stdLib.php
+++ b/inc/stdLib.php
@@ -117,6 +117,10 @@ function authuser($dbhost,$dbport,$dbuser,$dbpasswd,$dbname,$cookie) {
 function anmelden() {
 global $ERPNAME;
     ini_set("gc_maxlifetime","3600");
+       if (substr(getcwd(),-6)=="lxcars") {
+		chdir('..');
+		$lxcars = true;	
+	}
     if ( file_exists("../".$ERPNAME."/config/lx_office.conf") ) {
 	    $lxo = fopen("../".$ERPNAME."/config/lx_office.conf","r");
     } else if ( file_exists("../".$ERPNAME."/config/lx_office.conf.default") ) {
@@ -185,6 +189,9 @@ global $ERPNAME;
         $sql = "select * from defaults";
         $rs = $_SESSION["db"]->getAll($sql);
         $_SESSION["ERPver"]     = $rs[0]["version"];
+        if ($lxcars) {
+			chdir('lxcars');
+		}   
         return true;
     }
 }
-- 
1.7.5.4

